src/libraries/System.Threading.RateLimiting/src/System/Threading/RateLimiting/ConcurrencyLimiter.cs

```diff
@@ -39,6 +39,14 @@ public sealed class ConcurrencyLimiter : RateLimiter
         public ConcurrencyLimiter(ConcurrencyLimiterOptions options)
         {
             _options = options ?? throw new ArgumentNullException(nameof(options));
```

Comment:
Don't we need to copy the options class now? Otherwise the user may reuse the options for another limiter and potentially change its values.

Reply:
Yeah, good call.